src/main/java/com/blackducksoftware/integration/hub/alert/channel/hipchat/HipChatChannel.java

```diff
@@ -126,36 +107,10 @@ private int sendMessage(final HipChatConfigEntity config, final String apiUrl, f
 
     @Override
     public String testMessage(final HipChatConfigEntity config) {
-        final int responseCode = sendMessage(config, HIP_CHAT_API, "Test Message", HIP_CHAT_FROM_NAME + " Tester");
+        final int responseCode = sendMessage(config, HIP_CHAT_API, "Test Message", AlertConstants.ALERT_APPLICATION_NAME + " Tester");
         return String.valueOf(responseCode);
     }
 
-    private String createHtmlMessage(final ProjectData projectData) {
```

Comment:
I think we should leave the message formatting up to each channel for now rather than trying to abstract them for now

Reply:
I agree